docs/client/session.md

```diff
@@ -0,0 +1,25 @@
+# Semantic conventions for session
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+A session is collection of Logs, Events, and Spans associated with a specific device or user within a specific period of time. 
+A session is represented by a unique identifier that is attached to Logs, Events, and Spans as an attribute.
```

Comment:
We should include metrics too even though we are not planning on emitting metrics from Client-side apps. This statement could say a session is a collection of all the telemetry data emitted (Spans, Logs, Events and Metrics).

Reply:
Agreed, and I think we do probably have metrics use cases in mobile (count of slow frame renders, for instance, or other device metrics like memory usage or free space).

Reply:
Is there a use case for including session.id on metrics, given that the values are typically of high cardinality?

Reply:
Nope, I don't have a use case for including `session.id` on metrics. Seems like a problem.

Reply:
My intention was only to indicate that the metric was emitted in the context of a particular session;  I understand the concept of identifying vs describing attributes of a metric is still under consideration. Regarding the attribute being high cardinality, one option is to document this so that the backends could drop this metric attribute during aggregation.

However, since this looks more confusing than helpful, we can drop the idea of adding Session ID on metrics emitted.

Reply:
maybe a provision for metrics as exemplars having a session id?

Reply:
I think the use case is simply to know that a metric is emitted and associated with a particular session, as @scheler mentions. And I think the use case @breedx-splk brings up is perfect: I want to know how many dropped/slow-render frames there within a given sessions to identify its health (or whatever). We would need this association for that to happen in a reasonable way.

In terms of cardinality, I think the utility is to have the association, not using it as a means to group-by, so I think it's just fine.

Reply:
>  want to know how many dropped/slow-render frames there within a given sessions to identify its health (or whatever). We would need this association for that to happen in a reasonable way.

This is pretty much what a exemplar is https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exemplars.

If `session.id` is recorded on a span, and you have a metric, say `frames.render_time` with an exemplar, you could see a spike in render time, navigate to the trace and there you can see the session id. 

But one thing I'm not understanding, maybe I'm lacking context: What would knowing the sessionId help in such case (slow/dropped frames)? Wouldn't simply knowing that there's a slow down in frame rendering be enough to know there's a problem with the application? I would think, it would be valuable to know maybe which UI/Page it happened, which should be low cardinality and could be added to metrics. Sorry if I'm totally wrong here.